src/applications/check-in/day-of/pages/NextOfKin.jsx

```diff
@@ -26,6 +26,13 @@ const NextOfKin = props => {
   } = useFormRouting(router);
   const { setShouldSendDemographicsFlags } = useSessionStorage(false);
 
+  const { getCheckinComplete } = useSessionStorage(false);
```

Comment:
Could this happen in the ConfirmablePage component to avoid repeating this code for each demographic page?

Reply:
Similar to how you are doing it on the TravelPage component.